app/src/auto_validator/core/tests/conftest.py

```diff
@@ -42,3 +80,48 @@ def __eq__(self, other):
             return self.func(other)
 
     return EqualityMock
+
+
+@pytest.fixture
+def wallet():
+    coldkey_name = "auto-validator7"
+    hotkey_name = "testhotkey7"
+    command1 = f"btcli wallet new_coldkey --wallet.name {coldkey_name}"
```

Comment:
it'd be better to call proper functions from bittensor, but if it aint broken, don't fix it